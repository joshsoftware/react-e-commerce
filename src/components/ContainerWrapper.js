import React from 'react';
import { Container } from 'reactstrap';
import PropTypes from 'prop-types';

const ContainerWrapper = ({ styleClass, data, fluid }) => {
  return (
    <Container className={styleClass} fluid={fluid}>
      {data}
    </Container>
  );
};

export default ContainerWrapper;

ContainerWrapper.propTypes = {
  styleClass: PropTypes.string,
  data: PropTypes.oneOfType([PropTypes.element, PropTypes.array]),
  fluid: PropTypes.oneOfType([PropTypes.bool, PropTypes.string])
};
