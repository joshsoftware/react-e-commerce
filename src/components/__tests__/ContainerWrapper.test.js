import React from 'react';
import { render } from '@testing-library/react';
import ContainerWrapper from '../ContainerWrapper';

describe('ContainerWrapper Component', () => {
  it('Must return container', () => {
    const { asFragment } = render(<ContainerWrapper />);

    expect(asFragment()).toMatchSnapshot();
  });
});
