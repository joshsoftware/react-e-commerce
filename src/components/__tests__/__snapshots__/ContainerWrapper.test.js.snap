// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContainerWrapper Component Must return container 1`] = `
<DocumentFragment>
  <div
    class="container"
  />
</DocumentFragment>
`;
